### ISSUE TITLE
Expose core translation status

### ISSUE THREAD

**Issue by ro0NL** [symfony/symfony-docs#10417]
Based on `validators.en.xml`, `security.en.xml` etc. we can detect which translations are missing and which locales are "complete". Such a page could be automatically updated during a travis build step perhaps.

I think it improves contributing the missing pieces, and actually documents which locales are fully supported.

**Comment by javiereguiluz** [comment]
Good idea! We've created a little PHP script to display the status of the translations. We're polishing some things and after that, we'll create a pull request to add it to Symfony. Thanks.

**Comment by javiereguiluz** [comment]
The script mentioned above has been merged into Symfony (https://github.com/symfony/symfony/pull/30237) so I'm closing this as "fixed".

I know this is not exactly what you asked for ... but the script is a good first step. Thanks for understanding it.
